ivy/data_classes/array/experimental/losses.py

```diff
@@ -1,6 +1,52 @@
 # global
 import abc
+from typing import Optional, Union
+
+# local
+import ivy
 
 
 class _ArrayWithLossesExperimental(abc.ABC):
-    pass
+    def huber_loss(
+        self: ivy.Array,
+        pred: Union[ivy.Array, ivy.NativeArray],
+        /,
+        *,
+        reduction: str = "mean",
```

Comment:
```suggestion
        reduction: Optional[str] = "mean",
```

I believe this parameter should also be optional.